app/client/src/pages/reset-password.tsx

```diff
@@ -1,53 +1,56 @@
 import * as React from 'react';
-import { Grid, Paper } from '@material-ui/core';
+import { Typography, Avatar, Paper, Grid } from '@material-ui/core';
+import RefreshIcon from '@material-ui/icons/VpnKey';
 import { makeStyles } from '@material-ui/core/styles';
-
-import FadeThrough from '@local/animations/FadeThrough';
 import { useRouter } from 'next/router';
-// import PasswordResetForm from '@local/domains/Auth/PasswordResetForm';
-// TODO re-add password reset form
+import { ResetPasswordForm } from '@local/features/accounts/ResetPasswordForm';
 
 const useStyles = makeStyles((theme) => ({
     root: {
         width: '100%',
         height: '100%',
     },
     paper: {
-        padding: theme.spacing(2),
-        [theme.breakpoints.up('md')]: {
-            // boxShadow: theme.shadows[10],
-        },
-        [theme.breakpoints.down('sm')]: {
-            borderRadius: 0,
-        },
+        maxWidth: 425,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
```

Comment:
I think it might be best to take advantage of the `Grid` component here rather than using css.